src/base_utils/ffipp.js

```diff
@@ -1,6 +1,6 @@
 /// <reference types="./ffipp.d.ts"/>
 
-import "npm:reflect-metadata";
+import "https://esm.sh/reflect-metadata@0.2.2";
```

Comment:
is there a benefit in using esm.sh over standard npm?

Reply:
no.

There was some issue where the type checker was not being able to see the `Reflect.*` like `Reflect.defineMetadata` unless I imported from esm.sh. I think the actual solution would be to place the import somewhere and import it everywhere we use `Reflect.*`

Reply:
Should I revert that?

Reply:
No It's okay for now. but if you found a proper solution to have types using npm, open a new PR

Reply:
cool thanks for the review